.github/workflows/update-and-publish-helm-chart.yml

```diff
@@ -0,0 +1,44 @@
+name: update-and-publish-cromwell-helm-chart
+on:
+  push:
+    branches:
+      - 'main'
+
+jobs:
+  update-and-publish:
+    name: Update and publish new cromwell-helm chart
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repository
+        uses: actions/checkout@v2
+
+      - name: Find out next chart version
+        run: |
+          currentVersion=$(grep 'version:' cromwell-helm/Chart.yaml | awk '{print $NF}')
+
+          # this will increment only the last version digit. For example, it will update 0.1.10 => 0.1.11
+          nextVersion=$(echo ${currentVersion} | awk -F. -v OFS=. '{$NF += 1 ; print}')
```

Comment:
Question for reviewers: Do we want to keep updating only the patch version digit or should we move to updating the minor version digit? So go from 0.1.10 => 0.2.0 => 0.3.0?

Reply:
For now, this seems fine.

TOL for the future... I wonder if there's a way actions can be based on PRs merging rather than on `main` updating? If so, we could maybe parameterize the action with "type of version" and update the trigger to do something like "patch by default, minor or major if "[minor version] or [major version]" appear in the PR title?

Reply:
Or actually, even if we stay on "changes to main" we could use the most recent commit message in the same way

Reply:
I changed it from merging based on PRs to merges to main because the [auto update Cromwell version in helm chart Github action](https://github.com/broadinstitute/cromwell/pull/6739) makes a commit directly to the `main` branch. As a result if we keep it based on PRs, we miss out on updating chart version when Cromwell version update happens.

Reply:
I think we should update the minor version.

That's what I did in my PR before we put this PR in charge of that.

My reasoning is that a Cromwell upgrade should typically be a change that adds some features and is backwards-compatible.